src/replace/module.coffee

```diff
@@ -16,4 +19,16 @@ wrapIfNeeded = (fakeThing, realThing) ->
   if isConstructor(realThing)
     wrapWithConstructor(fakeThing)
   else
+    replaceProperties(fakeThing, realThing)
```

Comment:
Not sure if this is the proper place for this action, or if it introduces temporal coupling.